app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,24 @@
+module SessionsHelper
+    # Logs in the given user.
+    def log_in(user)
+        session[:user_id] = user.id
+    end
+
+    # Returns the current logged-in user (if any).
+    def current_user
+        if session[:user_id]
+            @current_user ||= User.find_by(id: session[:user_id])
+        end
+    end
+
+    # Returns true if the user is logged in, false otherwise.
+    def logged_in?
+        !current_user.nil?
```

Comment:
@kusanagi98 
Có thể dùng
```suggestion
        current_user.present?
```